### PR TITLE
Feature/schedule welcome message

### PR DESCRIPTION
**Pull request by FlorisE**
Implemented functionality for renewing welcome message on map change and on interval.

### CODE REVIEW

**Comment by FlorisE** [comment]
Hopefully it will look a bit more clean after the blacklist changes are merged, otherwise I can create a new pull request.

**Comment by MarechJ** [comment]
Ping me on discord once the blacklist one is merged I'll have a look at this one :)

File: rcon/extended_commands.py [hunk 1]
```diff
@@ -463,6 +461,16 @@ def set_maprotation(self, rotation):
 
         return [first] + rotation
 
+    def do_perma_ban_and_return_ban_log(self, player, reason):
```
Comment (MarechJ) [inline rcon/extended_commands.py:464]: It's a shame this MR is entangled with the ban one. I'll resume the review once we merge the ban one

**Comment by MarechJ** [comment]
We'd need to separate the scheduling code from the rest because it's not relevant anymore, reopen if you have the motivation to do so

**Pull request closed without merging**